repos.yaml

```diff
@@ -1313,3 +1313,10 @@ repositories:
         email: kubernetes-sig-ui@googlegroups.com
       - name: Cédric de Saint Martin
         email: cedric@desaintmartin.fr
+  - name: newrelic
+    url: https://helm-charts.newrelic.com
```

Comment:
The repository also needs to be added to the `config/repo-values.yaml` file.

Reply:
Fixed in 4b6fff3, thanks!